std/stdio.d

```diff
@@ -2808,6 +2808,80 @@ is empty, throws an `Exception`. In case of an I/O error throws
         assert(i == witness.length);
     }
 
+/**
+Returns an $(REF_ALTTEXT input range, isInputRange, std,range,primitives)
+set up to read from the file handle.
+
+The element type for the range will be `ubyte`. Range primitives
+may throw `StdioException` on I/O error.
+
+Example:
+---------
+void main()
+{
+    // Read standard input
+    foreach (ubyte c; stdin.asUbytesRange)
+    {
+        ... use c ...
+    }
+}
+---------
+
+The parameter may be a number (as shown in the example above) recommending the
+size of the buffer.
+
+Example:
+---------
+void main()
+{
+    // Read standard input
+    foreach (ubyte c; stdin.asUbytesRange(4096))
+    {
+        ... use c ...
+    }
+}
+---------
+
+With the mentioned limitations, `asUbytesRange` works with any algorithm
+compatible with input ranges.
+
+Returns: A range initialized with the `File` object.
+
+Asserts: Buffer size is non-zero. In case of an I/O error throws
+`StdioException`.
+ */
+    auto asUbytesRange(size_t bufferSize = 4096)
+    {
+        assert(bufferSize > 0);
+
+        import std.algorithm.iteration;
+        return byChunk(bufferSize).joiner;
```

Comment:
We generally try not to add "shallow" functions to Phobos. This is just a simple wrapper. It doesn't provide any new functionality, and IMHO, it's just too simple to add.